### PR TITLE
Enabling Clowder for  Cloudigrade

### PR DESCRIPTION
**Pull request by abellotti**
Clowderizing Cloudigrade supporting deployment to minikube as well as the ephemeral cluster.

TODO:
- [x] Add the Azure config and parameters
- [x] clowderize and add support for postigrade in minikube/ephemeral
- [x] re-test on ephemeral once https://github.com/RedHatInsights/clowder/pull/389 is merged.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -28,12 +28,20 @@ RUN microdnf update \
     && microdnf remove libcurl-devel gcc python3-devel openssl-devel annobin -y \
     && microdnf clean all
 
+# Need jq for parsing the clowder configuration json in shell scripts
+RUN curl -L https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 -o jq \
```
Comment (katherine-black) [inline Dockerfile:32]: Could we grab this from dnf/epel?

Comment (abellotti) [reply Dockerfile:32]: yep, microdnf ftw!  👍

Comment (katherine-black) [reply Dockerfile]: Thanks!

File: Dockerfile [hunk 2]
```diff
@@ -28,12 +28,18 @@ RUN microdnf update \
     && microdnf remove libcurl-devel gcc python3-devel openssl-devel annobin -y \
     && microdnf clean all
 
+# Need jq for parsing the clowder configuration json in shell scripts
+RUN microdnf install jq -y \
```
Comment (infinitewarp) [inline Dockerfile:32]: Any reason not to include this with the other packages above?

Comment (abellotti) [reply Dockerfile:32]: The dnf installs seemed to be categorized, second one postgres, third one python.  I could add it to the first group though, seems general enough.

**Review by katherine-black — APPROVED** [review]

**Pull request merged**